testing-modules/testing-assertions/src/test/java/com/baeldung/optional/OptionalUnitTest.java

```diff
@@ -0,0 +1,69 @@
+package com.baeldung.optional;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class OptionalUnitTest {
```

Comment:
By convention JUnit 5 tests don't have `public` modifiers on methods or the class

Reply:
yes that is true